openstack/compute/v2/extensions/remoteconsoles/requests.go

```diff
@@ -0,0 +1,83 @@
+package remoteconsoles
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// RemoteConsoleProtocol represents valid remote console protocol.
+// It can be used to create a remote console with one of the pre-defined protocol.
+type RemoteConsoleProtocol string
```

Comment:
Since the package name is `remoteconsoles`, I think we can shorten this to:

```go
type ConsoleProtocol string
```

And the name will still have a clear description of the type.

Reply:
Seems better, yeah.